test/extended/oauth/htpasswd.go

```diff
@@ -53,18 +54,17 @@ var _ = g.Describe("[Suite:openshift/oauth/htpasswd] HTPasswd IDP", func() {
 			MappingMethod:   "claim",
 			Provider:        *htpasswdProvider,
 		}
-		tokenReqOpts, cleanup, err := utiloauth.DeployOAuthServer(oc, []osinv1.IdentityProvider{htpasswdConfig}, nil, secrets)
+		newTokenReqOpts, cleanup, err := utiloauth.DeployOAuthServer(oc, []osinv1.IdentityProvider{htpasswdConfig}, nil, secrets)
 		defer cleanup()
 		o.Expect(err).ToNot(o.HaveOccurred())
+		tokenReqOpts := newTokenReqOpts("testuser", "password")
 		e2e.Logf("got the OAuth server address: %s", tokenReqOpts.Issuer)
-
-		token, err := utiloauth.RequestTokenForUser(tokenReqOpts, "testuser", "password")
```

Comment:
I think the `RequestTokenForUser` function should be deleted unless in favor of `newTokenReqOpts` but I have not checked to see if there are other usages.

Reply:
Done. Dropped.